Source/WebKit/UIProcess/WebLockRegistryProxy.h

```diff
@@ -30,27 +30,26 @@
 #include <WebCore/ScriptExecutionContextIdentifier.h>
 #include <WebCore/WebLockIdentifier.h>
 #include <WebCore/WebLockMode.h>
+#include <wtf/RefCounted.h>
 #include <wtf/TZoneMalloc.h>
 
-namespace WTF {
-template<typename T> struct IsDeprecatedWeakRefSmartPointerException;
-template<> struct IsDeprecatedWeakRefSmartPointerException<WebKit::WebLockRegistryProxy> : std::true_type { };
-}
-
 namespace WebCore {
 struct ClientOrigin;
 struct WebLockManagerSnapshot;
 }
 
 namespace WebKit {
 
-class WebLockRegistryProxy;
 struct SharedPreferencesForWebProcess;
 
-class WebLockRegistryProxy final : public IPC::MessageReceiver {
+class WebLockRegistryProxy final : public IPC::MessageReceiver, public RefCounted<WebLockRegistryProxy> {
```

Comment:
It would be better to forward to ref-counting to the owner WebProcessProxy.

Reply:
Fixed!